main.tf

```diff
@@ -54,15 +54,14 @@
  */
 
 module "dcos-bootstrap" {
-  source  = "dcos-terraform/dcos-core/template"
```

Comment:
Needs set back to this source prior to merge.

Reply:
will do